AdminWebsite/AdminWebsite/ClientApp/src/app/popups/wait-popup/wait-popup.component.spec.ts

```diff
@@ -16,10 +19,16 @@ describe('WaitPopupComponent', () => {
     beforeEach(() => {
         fixture = TestBed.createComponent(WaitPopupComponent);
         component = fixture.componentInstance;
+        de = fixture.debugElement;
+        headingText = de.query(By.css('.govuk-heading-m'));
         fixture.detectChanges();
     });
 
     it('should create wait pop up component', () => {
         expect(component).toBeTruthy();
     });
+
+    it('should have heading Confirming bookings can take up to 60 seconds to complete.', () => {
+        expect(headingText.nativeElement.innerText).toEqual('Confirming bookings can take up to 60 seconds to complete.');
```

Comment:
As above, I strongly believe we dont need to test the text content, I would rather not have this test at all or add an id or data handler to the element and assert that element exists